src/main/resources/edu/cornell/kfs/coa/businessobject/datadictionary/SubFundGroup.xml

```diff
@@ -37,277 +39,272 @@
-		<property name="inquirySections">
-			<list>
-				<ref bean="SubFundGroup-inquirySectionDefinition" />
-			</list>
-		</property>
-	</bean>
+        <property name="inquirySections">
+            <list>
+                <ref bean="SubFundGroup-inquirySectionDefinition"/>
+            </list>
+        </property>
+    </bean>
 
-	<bean id="SubFundGroup-inquirySectionDefinition" parent="SubFundGroup-inquirySectionDefinition-parentBean" />
-	<bean id="SubFundGroup-inquirySectionDefinition-parentBean"
-		parent="InquirySectionDefinition" abstract="true">
-		<property name="title" value="" />
-		<property name="numberOfColumns" value="1" />
-		<property name="inquiryFields">
-			<list>
-				<bean parent="FieldDefinition" p:attributeName="subFundGroupCode" />
-				<bean parent="FieldDefinition" p:attributeName="subFundGroupDescription" />
-        <bean parent="FieldDefinition" p:attributeName="extension.subFundGroupDetailedDescr"/>
-				<bean parent="FieldDefinition" p:attributeName="subFundGroupTypeCode" />
-				<bean parent="FieldDefinition" p:attributeName="fundGroupCode" />
-				<bean parent="FieldDefinition" p:attributeName="subFundGroupWagesIndicator" />
-				<bean parent="FieldDefinition" p:attributeName="financialReportingSortCode" />
-				<bean parent="FieldDefinition" p:attributeName="active" />
-				<bean parent="FieldDefinition" p:attributeName="fundGroupBudgetAdjustmentRestrictionLevelCode" />
-				<bean parent="FieldDefinition" p:attributeName="accountRestrictedStatusCode" />
-			</list>
-		</property>
-	</bean>
+    <bean id="SubFundGroup-sectionDefinition" parent="SubFundGroup-sectionDefinition-parentBean"/>
+    <bean abstract="true"
+          id="SubFundGroup-sectionDefinition-parentBean"
+          parent="sectionDefinition"
+    >
+        <property name="fields">
+            <list>
+                <ref bean="SubFundGroup-subFundGroupCode"/>
+                <ref bean="SubFundGroup-subFundGroupDescription"/>
+                <ref bean="SubFundGroup-extension-subFundGroupDetailedDescr"/>
+                <ref bean="SubFundGroup-subFundGroupTypeCode"/>
+                <ref bean="SubFundGroup-subFundGroupType.subFundGroupTypeDescription"/>
+                <ref bean="SubFundGroup-fundGroupCode"/>
+                <ref bean="SubFundGroup-fundGroup.name"/>
+                <ref bean="SubFundGroup-subFundGroupWagesIndicator"/>
+                <ref bean="SubFundGroup-financialReportingSortCode"/>
+                <ref bean="SubFundGroup-active"/>
+                <ref bean="SubFundGroup-fundGroupBudgetAdjustmentRestrictionLevelCode"/>
+                <ref bean="SubFundGroup-accountRestrictedStatusCode"/>
+            </list>
+        </property>
+    </bean>
 
-	<!-- Business Object Lookup Definition -->
+    <bean id="SubFundGroup-inquirySectionDefinition" parent="SubFundGroup-inquirySectionDefinition-parentBean"/>
+    <bean id="SubFundGroup-inquirySectionDefinition-parentBean" abstract="true" parent="InquirySectionDefinition"
```

Comment:
Our old version of this bean was setting "numberOfColumns" to 1. However, the newer type of "section" bean doesn't seem to have this property, so I am not certain whether it is worth keeping it on this legacy bean still.

Reply:
It looks like KualiCo removed the explicit numberOfColumns=1 setting over 5 years ago, so our bean setup must not have been updated to reflect that until now. (I also confirmed that the 01/21/2021 version of the bean does not have a setting for that property.) I'm fine with excluding "numberOfColumns" from this bean.